database/src/main/resources/db/migration/V1.98.0__RHCLOUD-33635_add_constraint_integrations_unique_name.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE endpoints ADD CONSTRAINT endpoints_name_org_id_key UNIQUE ("name", org_id);
```

Comment:
We can't add a db constraint because many existing endpoints have already the same name in stage and prod.
A migration will be needed if we want add this constraint.